cypress/features/regression/inlineInputs.feature

```diff
@@ -31,14 +31,11 @@ Feature: InlineInputs component
       | <>                       |
 
   @positive
-  Scenario Outline: Change second inline input to <label>
-    When I set second inline input to "<label>"
-    Then second inline input on preview is "<label>"
+  Scenario Outline: Change second inline input to <input>
+    When I set second inline input to "<input>"
```

Comment:
improvement suggestion:
Steps:
```
 Scenario Outline: Change second inline input to <label>
    When I set 2nd inline input to "<label>"
    Then 2nd inline input on preview is "<label>"
```
Step defs:
```
When('I set {int}{word} inline input to {string}', (number, word, text) => {
  inlineInputs(number).clear().type(text);
  pressTABKey(); //to release focus
});

Then('{int}{word} inline input on preview is {string}', (number, word, text) => {
  inlineInputs(number).should('have.value', text);
});
```
Locators:
```
export const inlineInputs = index => cy.iFrame(INLINE_INPUTS)
  .eq(`${index - 1}`);

export const INLINE_INPUTS = 'input';
```
hint: locator `export const INLINE_INPUTS = 'input[data-element="input"]';` not working for me so I suggest to changed it
hint2: adjust input data per scenario

Reply:
fixed